ci/jenkins/README.md

```diff
@@ -29,6 +30,21 @@ should be deleted. This ensures that all tests are run on a clean testbed.
 * jenkins jobs validator: this job only executes for PRs that include changes to
   [ci/jenkins/jobs](/ci/jenkins/jobs). It validates the syntax of the jenkins jobs'
   configuration.
+* [EKS conformance/network policy](http://jenkins.antrea-ci.rocks/view/cloud/job/cloud-antrea-eks-conformance-test-native/)
+  community tests on EKS cluster using sonobuoy, focusing on "Conformance" and "Feature:NetworkPolicy", skipping the same regexes as in job __conformance__ above, as well as "NodePort" (See #690). <br>
+  Current test environment matrix:
+  
+  |  Run frequency  | K8s version  | Node OS type    |
+  | :-------------: | :----------: | :-------------: |
+  | Every other day | 1.15         | EC2 t3.medium   |
```

Comment:
`EC2 t3.medium` is not an OS, maybe rename this column `Node type` and add the appropriate `AMI` column?

Reply:
Done